code/modules/vending/kinkmate.dm

```diff
@@ -40,8 +40,8 @@
 				/obj/item/clothing/under/pants/chaps = 5
 				)
 	refill_canister = /obj/item/vending_refill/kink
-	default_price = 80
-	extra_price = 250
+	default_price = 5
+	extra_price = 15
```

Comment:
Same prices as the clothesmate, no matter what.

Reply:
I will not be judging on kinkmate. Kinkmate appeals to assistants who have very tight paychecks and putting ERP items out of their budget, preventing them from fucking off to dorms because of an arbitrary economy feature, isnt acceptable.

Reply:
I have no plan to make the server any more appealing to people who merely wish to wind up in a rendezvous scene, and this vendor is not strictly necessary for that either.

On top of that, those players usually do little beside sitting at the bar, typing stuff and getting an article of clothing they couldn't have from the loadout system, they'll likely save up a hundred or two for that anyway.

Reply:
I am not interested in a system with the purpose of being vindictive to ERPers by withholding them flavor / fetish items due to some kind of code grudge. There will be no change.

Reply:
It seems we are reached a deadlock then. I'm off to make my own PR, no gain in being stubborn.